### DISCUSSION TITLE
pico-littlefs-usb to STM32 project or fork

### DISCUSSION THREAD

**Discussion by vtjballeng** [oyama/pico-littlefs-usb#13] [Ideas]
Hi @oyama , I think this is a great solution! I have an STM32F4 based project using FatFS and I'd like to switch to littlefs. I haven't found any STM32 based solutions like yours that restores typical Windows USB like file system functionality. 

I'm curious if you'd consider an STM32F4 or similar project / fork or assist in a project for that purpose.

**Comment by oyama** [comment]
Thank you for your interest.

I have no experience working with STM devices, so I would recommend you to fork or start a similar project. However, my implementation of this is quite slow and restrictive; I have recently started a more straightforward scheme using RAM disks, so please refer to that as well:
https://github.com/oyama/pico-drive-sync
